web/src/app/pages/edit-survey/edit-survey.component.html

```diff
@@ -14,4 +14,40 @@
   limitations under the License.
 -->
 
-<h1 id="temp-title">Edit survey {{ (activeSurvey$ | async)?.id }}</h1>
+<div class="page">
+  <ground-header> </ground-header>
+
+  <div class="container">
+    <mat-nav-list class="left-menu">
+      <a mat-list-item id="survey-button" [routerLink]="['./survey']">
+        <mat-icon class="material-symbols-outlined button-icon"
+          >content_paste</mat-icon
```

Comment:
Nit: This html formatting is a little funky -- is there a reason for this?

Reply:
This was what the formatter gave me, I made `content_paste` its own line, now it looks better.